From af67380ed5206c02bfbecaf5abf3790f4b37f60f Mon Sep 17 00:00:00 2001
From: Ista Zahn <istazahn@gmail.com>
Date: Sun, 17 Oct 2021 14:20:08 -0400
Subject: [PATCH] loosen version pins

---
 tools/requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/requirements.txt b/tools/requirements.txt
index 5bf4784..77b8eac 100644
--- a/tools/requirements.txt
+++ b/tools/requirements.txt
@@ -1,2 +1,2 @@
-cython==0.29.20
+cython>=0.29.20
 torch>=1.5.0
-- 
2.39.2 (Apple Git-143)

